test/module/irohad/consensus/yac/yac_gate_test.cpp

```diff
@@ -175,8 +187,11 @@ TEST_F(YacGateTest, LoadBlockWhenDifferentCommit) {
   EXPECT_CALL(*hash_gate, on_commit()).WillOnce(Return(expected_commit));
 
   // convert yac hash to model hash
+  auto old_hash =
+      (*std::unique_ptr<iroha::model::Block>(expected_block->makeOldModel()))
+          .hash;
   EXPECT_CALL(*hash_provider, toModelHash(expected_hash))
-      .WillOnce(Return(old_expected_block.hash));
+      .WillOnce(Return(old_hash));
```

Comment:
expected_block->hash()

Reply:
`toModelHash()` uses old model

Reply:
#1116 fixes it

Reply:
> Depends on #1052

Looks like circular dependency

Reply:
@neewy We have to close this PR first, because #1116 was branched from this one, so it is ok